README.md

```diff
@@ -137,10 +166,15 @@ Note: `query` and `operationName` parameters are encoded as raw strings in the q
 
 ## POST
 
-A standard GraphQL POST request should have a body which contains values of the request parameters encoded according to
-the value of `Content-Type` header.
+A GraphQL POST request instructs the server to perform a query, mutation or subscription operation. A GraphQL POST request should have a body which contains values of the request parameters encoded according to the value of `Content-Type` header of the request.
+
+When sending a POST request to a GraphQL server a client MUST include a body.
```

Comment:
This line seems like it might be contradictory to the one above it.  The first one says `A GraphQL POST request should have a body...`, but this one says MUST.  Perhaps they could be combined?